components/modal/style/index.tsx

```diff
@@ -103,9 +101,16 @@ export const genModalMaskStyle: GenerateStyle<TokenWithCommonCls<AliasToken>> =
 
         [`${componentCls}-wrap`]: {
           ...box('fixed'),
+          zIndex: token.zIndexPopupBase,
           overflow: 'auto',
           outline: 0,
           WebkitOverflowScrolling: 'touch',
+
+          // Note: Firefox not support `:has` yet
+          [`&:has(${componentCls}${antCls}-zoom-enter), &:has(${componentCls}${antCls}-zoom-appear)`]:
```

Comment:
```suggestion
          [`&${componentCls}${antCls}-zoom-enter, &${componentCls}${antCls}-zoom-appear`]:
```
这里是不是等价这么写了？都是同一个元素上的选择器